### PR TITLE
EPL1.0 -> EPL2.0 and contribute.md

### PR DESCRIPTION
**Pull request by svenefftinge**
This PR changes the code license from EPL 1.0 to EPL 2.0 and adds a Contribute.md file.

see https://github.com/eclipse/xtext/issues/1176

### CODE REVIEW

**Comment by dhuebner** [comment]
@cdietrich I also noticed that in the other smaller repo but not sure if maybe the URL change is enough

**Comment by cdietrich** [comment]
@svenefftinge please check test failure 
See also my remark in core regarding settings

File: org.eclipse.xtext.activities/plugin.properties [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ###############################################################################
 # Copyright (c) 2008, 2016 itemis AG (http://www.itemis.eu) and others.
 # All rights reserved. This program and the accompanying materials
```
Comment (szarnekow) [inline org.eclipse.xtext.activities/plugin.properties]: The header of the properties files needs an update

File: org.eclipse.xtext.builder/schema/participant.exsd [hunk 2]
```diff
@@ -148,9 +148,9 @@ implementor of &lt;samp&gt;org.eclipse.xtext.builder.IXtextBuilderParticipant&lt
       <documentation>
          Copyright (c) 2010 itemis AG (http://www.itemis.eu) and others.
 All rights reserved. This program and the accompanying materials
-are made available under the terms of the Eclipse Public License v1.0
+are made available under the terms of the Eclipse Public License v2.0
```
Comment (szarnekow) [inline org.eclipse.xtext.builder/schema/participant.exsd]: the extension point schema has an outdated license header

File: org.eclipse.xtext.eclipse.tests/.settings/org.eclipse.jdt.ui.prefs [hunk 3]
```diff
@@ -70,7 +70,7 @@ org.eclipse.jdt.ui.keywordthis=false
...
...catchblock" name\="catchblock">// ${todo} Auto-generated catch block\n${exception_var}.printStackTrace();</template><template autoinsert\="true" context\="methodbody_context" deleted\="false" description\="Code in created function stubs" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.methodbody" name\="methodbody">// ${todo} Auto-generated function stub\n${body_statement}</template><template autoinsert\="true" context\="constructorbody_context" deleted\="false" description\="Code in created constructor stubs" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.constructorbody" name\="constructorbody">${body_statement}\n// ${todo} Auto-generated constructor stub</template><template autoinsert\="true" context\="getterbody_context" deleted\="false" description\="Code in created getters" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.getterbody" name\="getterbody">return ${field};</template><template autoinsert\="true" context\="setterbody_context" deleted\="false" description\="Code in created setters" enabled\="true" id\="org.eclipse.wst.jsdt.ui.text.codetemplates.setterbody" name\="setterbody">${field} \= ${param};</template></templates>
```
Comment (szarnekow) [inline org.eclipse.xtext.eclipse.tests/.settings/org.eclipse.jdt.ui.prefs]: template text is still pointing to epl 1.0

**Comment by szarnekow** [comment]
@cdietrich please have another look

**Pull request merged**